packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareManageNamedVersions.tsx

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useState } from "react";
+import { IModelApp } from "@itwin/core-frontend";
+import "./styles/ComparisonJobWidget.scss";
+
+
+export interface ManageNamedVersionsProps {
+  manageNamedVersionHref?: string;
+  onclickManageNamedVersion?: () => Promise<void>;
+  manageNamedVersionLabel?: string;
+}
+
+/**
+ * Provides a href that will, on a click, navigate to the provided link or invoke the provided onClick method.
+ *
+ * Please note if href and both on click are provided; the component will not use on click but will use href instead.
+ *
+ * ManageNamedVersionLabel will default to `Manage named versions` if not provided.
+ */
+export function ManageNamedVersions(props: ManageNamedVersionsProps) {
+  const [hasBeenClicked, setHasBeenClicked] = useState<boolean>();
```

Comment:
Why do you need to change the colour of the text on click? It will revert back anyway after the dialog is reopened and the state is reset.

Reply:
That's correct, but currently, when you click the text. It does not show any indication of being clicked. This is mostly for if the onclick function is provided, but href is not so the user can have UI feedback. Also, when clicking a URL, it usually turns purple after it has been "visited".